src/main/java/uk/gov/hmcts/darts/common/entity/DailyListEntity.java

```diff
@@ -52,42 +50,37 @@ public class DailyList {
 
     @ManyToOne
     @JoinColumn(name = COURTHOUSE_ID)
-    private Courthouse courtHouse;
+    private Courthouse courthouse;
 
     @Column(name = DAILY_LIST_OBJECT_ID)
-    private String dailyListObjectId;
+    private String legacyObjectId;
 
     @Column(name = UNIQUE_ID)
     private String uniqueId;
 
     @Column(name = JOB_STATUS)
-    private String jobStatus;
+    private String status;
 
     @Column(name = TIMESTAMP)
-    private OffsetDateTime timeStamp;
+    private OffsetDateTime timestamp;
 
     @Column(name = DAILY_LIST_ID)
-    private Integer dailylistId;
+    private Integer dailyListId;
 
     @Column(name = START_DATE)
-    private OffsetDateTime startDate;
+    private Date startDate;
```

Comment:
Are we saying these 2 columns aren't `TIMESTAMP WITH TIMEZONE`? If not, seems wrong to change the Java type here

If they are to become simply `DATE` types in postgres then I think they should be `LocalDate` Java types here?